yarn.lock

```diff
@@ -2,33 +2,37 @@
 # yarn lockfile v1
 
 
-"@celo/base@5.0.4", "@celo/base@^5.0.4":
```

Comment:
there are changes to this file because PRs renovate merged without rebuilding the yarn.lock?

Reply:
Most likely because the 6.0.0 version of `@celo/base` was recently released, so it updated during rebuild